user_tools/src/spark_rapids_pytools/rapids/tool_ctxt.py

```diff
@@ -87,7 +87,7 @@ def get_deploy_mode(self) -> Any:
         return self.platform_opts.get('deployMode')
 
     def is_fatwheel_mode(self) -> bool:
-        return self.get_ctxt('fatwheelModeEnabled')
+        return self.get_ctxt('fatWheelModeEnabled')
```

Comment:
do you want to rename the function name to `is_fat_wheel_mode` as well to be consistent?

Reply:
Renamed to `is_fat_wheel_mode`